docs/jwst/firstframe/description.rst

```diff
@@ -0,0 +1,8 @@
+
+Description
+===========
+
+The first frame correction step flags the first group as bad (the GROUP data
+quality flags for the first group in all integrations are reset to DO_NOT_USE),
```

Comment:
Technically the DQ flags are not "reset" to DO_NOT_USE, but rather the DO_NOT_USE flag value is added to or propagated into any existing flag values.

Reply:
Done.